cmd/fyne/internal/commands/serve.go

```diff
@@ -0,0 +1,129 @@
+package commands
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/urfave/cli/v2"
+
+	"fyne.io/fyne/v2/cmd/fyne/internal/util"
+)
+
+// Server serve fyne wasm application over http
+type Server struct {
+	port              int
+	srcDir, icon, dir string
+}
+
+// Serve return the cli command for serving fyne wasm application over http
+func Serve() *cli.Command {
+	s := &Server{}
+
+	return &cli.Command{
+		Name:        "serve",
+		Usage:       "Package an application using WebAssembly and expose it via a web server.",
+		Description: `The serve command packages an application using WebAssembly and expose it via a web server which port can be overridden with port.`,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "sourceDir",
+				Usage:       "The directory to package, if executable is not set",
+				Value:       "",
+				Destination: &s.srcDir,
+			},
+			&cli.StringFlag{
+				Name:        "icon",
+				Usage:       "The name of the application icon file.",
+				Value:       "Icon.png",
+				Destination: &s.icon,
+			},
+			&cli.IntFlag{
+				Name:        "port",
+				Usage:       "The port to have the http server listen on (default: 8080).",
+				Value:       8080,
+				Destination: &s.port,
+			},
+		},
+		Action: s.Server,
+	}
+}
+
+// AddFlags adds the flags for interacting with the Installer.
+//
+// Deprecated: Access to the individual cli commands are being removed.
+func (s *Server) AddFlags() {
+	flag.IntVar(&s.port, "port", 8080, "The port to have the http server listen on (default: 8080).")
+	flag.StringVar(&s.srcDir, "sourceDir", "", "The directory to package, if executable is not set")
+	flag.StringVar(&s.icon, "icon", "Icon.png", "The name of the application icon file")
+}
+
+// PrintHelp prints the help for the install command.
+//
+// Deprecated: Access to the individual cli commands are being removed.
+func (s *Server) PrintHelp(indent string) {
+	fmt.Println(indent, "The serve command setup a local http server that is necessary to test WebAssembly package.")
+	fmt.Println(indent, "Command usage: fyne serve [parameters]")
+}
```

Comment:
I don't think you really need to add these methods. These deprecation notices are for the older commands that were moved out to the internal commands package, hince this should be removed when the new API is removed from there.